stage1/init/common/pod.go

```diff
@@ -155,7 +155,7 @@ func WritePrepareAppTemplate(p *stage1commontypes.Pod) error {
 	return nil
 }
 
-func writeAppReaper(p *stage1commontypes.Pod, appName string) error {
+func writeAppReaper(p *stage1commontypes.Pod, appName string, appRootDirectory string, appExecStartString string) error {
```

Comment:
The parameter should not be named "appExecStartString" anymore since only passing the binary path is passed now.